tests/rollbacks/test_soaconfigs.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from sticht.rollbacks.soaconfigs import get_cluster_from_soaconfigs_filename
+from sticht.rollbacks.soaconfigs import get_rollback_files_from_soaconfigs
+
+
+@pytest.mark.parametrize(
+    'dir_files, expected_files', (
+        # single service with a single rollback file
+        (
+            (
+                'serviceA/kubernetes-test.yaml',
+                'serviceA/slo-test.yaml',
+                'serviceA/rollback-test.yaml',
+            ),
+            ('serviceA/rollback-test.yaml',),
+        ),
+        # multiple services with a single rollback file
+        (
+            (
+                'serviceA/kubernetes-test.yaml',
+                'serviceA/slo-test.yaml',
+                'serviceA/rollback-test.yaml',
+                'serviceB/rollback-test.yaml',
+            ),
+            ('serviceA/rollback-test.yaml', 'serviceB/rollback-test.yaml'),
+        ),
+        # single service with multiple rollbacks
+        (
+            (
+                'serviceA/rollback-test.yaml',
+                'serviceA/rollback-test2.yaml',
+            ),
+            ('serviceA/rollback-test.yaml', 'serviceA/rollback-test2.yaml'),
+        ),
+        # file with an incorrect rollback prefix
+        (
+            (
+                'serviceA/rollbacknot-test.yaml',
+                'serviceA/rollbacknope-test2.yaml',
+            ),
+            tuple(),
+        ),
+        # empty directory
```

Comment:
Can we also test for services that would like to add them in a rollback directory? I know some services have slo directories as well.

Reply:
those are for something slightly different - the slo-based rollback mechanism used those folders to namespace slo by instance/endpoints/etc

it's probably not worth carrying support over for that since the configuration we care about will only be used for rollbacks - not rollbacks and general monitoring

Reply:
i ended up prohibiting this